pkg/components/openebs-operator/component.go

```diff
@@ -50,17 +61,22 @@ func (c *component) LoadConfig(configBody *hcl.Body, evalContext *hcl.EvalContex
 }
 
 func (c *component) RenderManifests() (map[string]string, error) {
-	tmpl, err := template.New("installer").Parse(operatorInstallerTmpl)
+	helmChart, err := util.LoadChartFromAssets(fmt.Sprintf("/components/openebs"))
```

Comment:
Why `fmt.Sprintf` here? Also, if we deal with paths, shouldn't we use [filepath.Join()](https://golang.org/pkg/path/filepath/#Join) I think.

Reply:
Removed the sprintf. The sprint was left over from copying from other component ;-)